code-challenges.test.js

```diff
@@ -48,9 +183,11 @@ const studyMinutesWeek2 = {
   thursday: 20,
   friday: 15,
   saturday: 65
-}
-// Expected output: [10, 15, 20, 45, 60, 65, 100]
+};
 
-// b) Create the function that makes the test pass.
+const combinedObjects = [studyMinutesWeek1, studyMinutesWeek2];
 
-// Pseudo code:
+const sortedValues = combineAndSortObjects(combinedObjects);
+console.log(sortedValues);
+
```

Comment:
Remember that the test should call your function. If that isn't happening, it would be good to go back and look at your test.